platform/security/core/security-core-impl/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java

```diff
@@ -171,18 +173,9 @@ public LogoutResponse buildLogoutResponse(String issuerOrEntityId, String status
   @Override
   public LogoutResponse buildLogoutResponse(
       String issuerOrEntityId, String statusCodeValue, String inResponseTo) {
-    return buildLogoutResponse(
-        issuerOrEntityId, statusCodeValue, inResponseTo, UUID.randomUUID().toString());
-  }
-
-  public LogoutResponse buildLogoutResponse(
-      String issuerOrEntityId, String statusCodeValue, String inResponseTo, String id) {
     if (issuerOrEntityId == null) {
```

Comment:
✏️ Outside the scope of this change, but related...could replace this with
```java
    Validate.notNull(issuerOrEntityId, "Issuer cannot be null");
```
(`org.apache.commons.lang.Validate` is in commons-lang which is already available to this bundle)